src/d/a/d_a_base.cpp

```diff
@@ -1,12 +1,18 @@
 #include "d/a/d_a_base.h"
 
+#include "c/c_lib.h"
+#include "common.h"
 #include "d/a/d_a_player.h"
 #include "d/a/obj/d_a_obj_base.h"
+#include "d/col/bg/d_bg_s.h"
 #include "d/d_heap.h"
 #include "d/d_room.h"
 #include "d/d_sc_game.h"
 #include "d/d_stage.h"
 #include "d/flag/enemyflag_manager.h"
+#include "d\col\bg\d_bg_s_gnd_chk.h"
```

Comment:
I guess wibo makes this work regardless but
```suggestion
#include "d/col/bg/d_bg_s_gnd_chk.h"
```